ntc_templates/templates/fortinet_get_system_interface_physical.textfsm

```diff
@@ -25,5 +26,7 @@ Start
   ^\s+status:\s+${STATUS}$$
   ^\s+speed:\s+${SPEED}(|Mbps)\s+\(Duplex:\s+${DUPLEX}\)$$ -> Record
   ^\s+speed:\s+${SPEED}$$ -> Record
+  ^\s+FEC:.*$$
+  ^\s+FEC_cap:.*$$
```

Comment:
It's possible to consolidate the two lines since this is a simple regex.
Click the "commit suggestion" button to merge this suggestion in. Thank you!

```suggestion
  ^\s+FEC(_cap)?:.*$$
```